package.json

```diff
@@ -72,12 +74,14 @@
   "scripts": {
     "build:evergreen": "node .evergreen/generate_evergreen_tasks.js",
     "build:ts": "gulp -f test/tools/ts-pipeline.js",
-    "check:atlas": "node test/tools/atlas_connectivity_tests.js",
     "check:bench": "node test/benchmarks/driverBench",
     "check:coverage": "nyc mocha --timeout 60000 --recursive test/functional test/unit",
-    "check:lint": "eslint -v && eslint index.js lib test",
+    "check:lint": "eslint -v && eslint src/**/*.ts test",
     "check:test": "mocha --recursive test/functional test/unit",
     "check:ts": "tsc --noEmit",
+    "check:atlas": "mocha --config \"test/manual/mocharc.json\" test/manual/atlas_connectivity.test.js",
```

Comment:
I didn't work too hard at this, it might be possible to specify a `mocharc.json` just for this folder, overriding the top level, but it didn't immediately work so I moved on.

Reply:
If you add `"extends": "../../mocharc.json",` to the config in the tests/manual folder it should work by only using ts-node for the tests in this directory, Apparently the config file is handled by yargs, so whatever it supports will work for us here.

Reply:
see I don't want it to extend the existing config, I want this one to effectively be the root config. I tried adding `"root": true` for a `.mocharc.json` in this folder, but for whatever reason it was picking up the top-level one first. I'd prefer to punt on this until after we get this PR merged, given the complexity of the change